doc-content/enterprise-only/admin-and-config/error-types-and-filters-ref.adoc

```diff
@@ -9,9 +9,9 @@ Error categorization and filtering is based on `ExecutionErrorFilters`. This int
 * `filter`: where the actual filtering, handling, and so on happens.
 * `getPriority`: indicates the priority that is used when calling filters.
 
-As only one filter can process given error, filters use a priority system to avoid having multiple filters returning alternative “views” of the same error. Priority allows more specialized filters to see if the error can be accepted, or otherwise allow another filter to handle it.
+As only one filter can process given error, filters use a priority system to avoid having multiple filters returning alternative “views” of the same error. Priority enables more specialized filters to see if the error can be accepted, or otherwise allow another filter to handle it.
```

Comment:
Filters process one error at a time and use a priority system to.....

Reply:
Done